addons/mail/static/src/core/thread_service.js

```diff
@@ -1068,7 +1099,7 @@ export class ThreadService {
     /**
      * @param {number} threadId
      * @param {string} data base64 representation of the binary
-     * @returns 
+     * @returns
```

Comment:
Could even remove this useless `@returns` if nothing relevant is returned

Reply:
and remove the return instruction in favor of just await the rpc